.github/workflows/build.yaml

```diff
@@ -0,0 +1,17 @@
+name: Build
+on: [push]
+
+jobs:
+  build:
+    name: Build SemTK
+    runs-on: ubuntu-latest
```

Comment:
It's a bummer that ubuntu-latest is still ubuntu-18.04, not ubuntu-20.04.  It doesn't matter here, but it matters over in RACK since the images are based on ubuntu-20.04 so I use ubuntu-20.04 in the CI workflow there too.

Reply:
Sounds like no action needed here for now, then?

Reply:
Right, I'm leaving ubuntu-latest vs. ubuntu-20.04 up to your choice (or Paul's).  I don't know why GitHub is taking so long to make ubuntu-latest the same as ubuntu-20.04 - it's been 6 months since 20.04 LTS came out.